tendrl/commons/objects/definition/master.yaml

```diff
@@ -21,6 +21,20 @@ namespace.tendrl:
     ceph-provisioner: "provisioner/ceph"
     ceph-installer: "provisioner/ceph"
   flows:
+    ConfigureMonitoring:
+      tags:
+        - "tendrl/node_$NodeContext.node_id"
```

Comment:
It is essential to make sure that the node is part of a cluster to be able to execute this flow.
So anyother tag required here?
certainly tag with provisioner contained in it might not be applicable as there is nothing that restricts this flow to be executed only on a provisioner node..

@r0h4n @shtripat

Reply:
@anmolbabu you may opt for `tendrl/integration/{node_id}` as this is added by `gluster-integration` for the node. @r0h4n ?

Reply:
tendrl/node_$id  is a valid and accurate tag. `tendrl/integration/$node_id` should be used to target jobs to gluster-integration (job type "sds")